source/developer/interactive-dialogs.rst

```diff
@@ -242,7 +242,18 @@ Moreover, Mattermost also allows the integration itself to perform input validat
 
   {"errors": {"num_between_0_and_10": "Enter a number between 0 and 10."}}
 
-Finally, once the request is submitted, we recommend the integration to respond with a system message or an ephemeral message confirming the submission.
+Finally, once the request is submitted, we recommend the integration to respond with a system message or an ephemeral message confirming the submission. This should be a separate request back to Mattermost once the service has received (and responded to) a submission request from a dialog:
+
+.. code-block:: none
```

Comment:
This code example was about sending dialog submission reply and this is unrelated to the above topic. Would you please remove it?

Reply:
Removed

Reply:
The change did seem to receive positive feedback from original reporter, so maybe we could keep it but describe it differently?

```
The amendment to the documentation has made it clearer about what the integration needs to do. If possible, a JSON example for the HTTP API (and endpoint) or a reference to the /posts/ephemeral documentation would be perfect.
```

Reply:
Let's just link to the API endpoint.